style.css

```diff
@@ -55,23 +146,189 @@ ul {
   color: #172b4d;
 }
 
+.tertiary_color {
+  color: #3a4a42;
+}
+
+.quaternary_color {
+  background-color: #ebf0ee;
+}
+
+/* Buttons */
+
+.btn1 {
+  background-color: #36b37e;
+  height: 48px;
+  width: 122px;
+  border-radius: 4px;
+  padding: 12px;
+  border: none;
+  cursor: pointer;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.btn1:enabled {
+  background-color: #36b37e;
+}
+
+.btn1:hover {
+  background-color: #36b37f;
+}
+
+.btn1:active {
+  background-color: #008552;
+}
+
+.btn_resume {
+  background-color: #36b37e;
+  width: 159px;
+}
+
+/* Classes */
+
+.article_about_me p {
+  margin-bottom: 35px;
+  border-radius: 8px;
+  height: 474px;
+  width: 327px;
+}
+
+.article_card {
+  border-radius: 8px;
+  height: 352px;
+  width: 327px;
+}
+
+.article_content {
+  border-radius: 8px;
+  height: 222px;
+  padding: 16px;
+  width: 327px;
+}
+
+.article_works {
+  border-radius: 8px;
+  height: 474px;
+  width: 327px;
+}
+
+.border {
+  border: 1px solid #d6d4d4;
+  border-radius: 5px;
+}
```

Comment:
- **_[OPTIONAL]_** Well done adding the correct background to your about section cards 🤝🏽, and while we are here please take the time to add the correct background to your list of languages, skills, and frameworks as you can see in the design 🛠.

### Your current design
![image](https://user-images.githubusercontent.com/43172164/206214302-da6523eb-9753-499c-8a2e-01bf409518a8.png)

### Improvements you can make
![image](https://user-images.githubusercontent.com/43172164/206214617-96c78802-1977-497f-9c86-49b159a35c40.png)